README.rst

```diff
@@ -32,6 +32,17 @@ to support modern computational graph libraries such as PyTorch and
 TensorFlow in order to make use of features such as autodifferentiation
 and GPU acceleration.
 
+Try out now with Pyolite
+------------------------
+
```

Comment:
This will currently render with nothing below it on GitHub's preview and on PyPI as mentioned in the first concern in https://github.com/scikit-hep/pyhf/issues/1775#issuecomment-1076791563.